src/content/app/genome-browser/components/track-settings-panel/track-settings-views/TranscriptTrackSettings.tsx

```diff
@@ -0,0 +1,105 @@
+/**
```

Comment:
What about this file? Is it used anywhere?

Reply:
This is not used anywhere atm. It's for future, delete?

Reply:
deleted